sys/gnu.cmake

```diff
@@ -70,7 +70,7 @@ set(C_FLAGS_COVERAGE "-O0 -g --coverage")
 
 # LAPACK and BLAS
 # (if the BLAS library contains the LAPACK functions, set LAPACK_LIBRARY to "NONE")
-#set(BLAS_LIBRARY "openblas" CACHE STRING "BLAS libraries to link")
+set(BLAS_LIBRARY "openblas" CACHE STRING "BLAS libraries to link")
```

Comment:
Don't commit changes to the toolchain file

Reply:
@fbonafe this seems to be the culprit for the CI fails, can you revert change to this line and squash commits?

Reply:
Didn't mean to, I'll revert it now, thanks for noticing!

Reply:
Just forced-push the same commit without the toolchain file